zulip_botserver/zulip_botserver/server.py

```diff
@@ -28,6 +29,32 @@ def read_config_section(parser: configparser.ConfigParser, section: str) -> Dict
     }
     return section_info
 
+def read_config_from_env_vars(bot_name: Optional[str] = None) -> Dict[str, Dict[str, str]]:
+    bots_config = {}  # type: Dict[str, Dict[str, str]]
+    env_config = OrderedDict()   # type: Dict[str, Dict[str, str]]
+    json_config = os.environ.get('ZULIP_BOTSERVER_CONFIG')   # type: Optional[str]
+
+    # Check for presence of environment variable otherwise throws appropriate errors
```

Comment:
I think we can drop this comment.  Comments are good for explaining the "why" of something, or for particularly gnarly algorithms.  This is neither.